decred/decred/dcr/dcrdata.py

```diff
@@ -1272,12 +1295,12 @@ def purchaseTickets(self, keysource, utxosource, req):
 
         txFeeIncrement = req.txFee
         if txFeeIncrement == 0:
-            txFeeIncrement = self.relayFee()
+            txFeeIncrement = relayFee
 
         # Send the split transaction.
         # sendOutputs takes the fee rate in atoms/byte
         splitTx, splitSpent, internalOutputs = self.sendOutputs(
-            splitOuts, keysource, utxosource, int(txFeeIncrement / 1000)
+            splitOuts, keysource, utxosource, txFeeIncrement // 1000, allowHighFees
```

Comment:
This seems like it's a bug. `sendOutputs` doesn't actually take fees in `atoms / kB`. Shouldn't be coverting here. We've been sending split transactions with low fees, I think.

Reply:
Looks that way.

Reply:
Here,s a ticket purchase from a month ago done with tinydecred, but fee looks ok https://testnet.dcrdata.org/tx/0db01ee2735021b979f0a19ca5deca62d1ef615daf63dca0b32be33c62cdabc2/out/32

Reply:
Ah, I see. I removed it but, this was being converted back to coins inside of sendOutputs.

Reply:
You can go to atoms/kB everywhere below the `AccountSettingsScreen` if you'd like.

Reply:
I guess we could go all atoms/byte. I'll make an issue about it later. I think they are all in atoms/kB below the AccountSettingsScreen currently.